source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -158,10 +162,16 @@ def create_next_kraken_request(request, responses):
     """
     one_minute = 60
     if request['clockwise']:
-        soonest_departure = min(j.departure_date_time for r in responses for j in r.journeys)
+        try:
+            soonest_departure = min(j.departure_date_time for r in responses for j in r.journeys if _has_pt(j))
```

Comment:
If there is only non pt solution we won't do any next call (is it bad?) but we will raise some error since soonest_departure won't be define when calculating new_datetime